test/units/module_utils/common/arg_spec/test_aliases.py

```diff
@@ -95,11 +95,11 @@ def test_aliases(arg_spec, parameters, expected, deprecation, warning):
     assert isinstance(result, ValidationResult)
     assert result.validated_parameters == expected
     assert result.error_messages == []
-    assert result._aliases == {
-        alias: param
+    assert result._aliases == dict([
```

Comment:
This could be written without the explicit list. Passing the expression directly to `dict` should suffice.

Reply:
Even better. Love it. Thank you for the tips.